Mathlib/MeasureTheory/Measure/Haar/Unique.lean

```diff
@@ -263,16 +276,23 @@ function `f`. -/
 @[to_additive "Given two left-invariant measures which are finite on compacts,
 `addHaarScalarFactor μ' μ` is a scalar such that `∫ f dμ' = (addHaarScalarFactor μ' μ) ∫ f dμ` for
 any compactly supported continuous function `f`."]
-noncomputable def haarScalarFactor (μ' μ : Measure G) [IsFiniteMeasureOnCompacts μ]
```

Comment:
why `noncomputable`? It should be possible to prove that the factor does not depend on the choice of the chosen functions…

Reply:
`noncomputable` means that I'm not providing an algorithm by which Lean's kernel could evaluate the number. Since real numbers are already noncomputable by definition, there is no way this function could be computable.